upcoming_changes/2975.doc.rst

```diff
@@ -0,0 +1 @@
+Add a note in the user guide to explain that when a file contains several dataset, :py:func:`~.io.load` returns a list of signals instead of a single signal and that list indexation can be used to access single signal.
```

Comment:
```suggestion
Add a note in the user guide to explain that when a file contains several datasets, :py:func:`~.io.load` returns a list of signals instead of a single signal and that list indexation can be used to access a single signal.
```